content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Monitoring Page Performance
+kind: documentation
+further_reading:
+  - link: "https://www.datadoghq.com/blog/real-user-monitoring-with-datadog/"
+    tag: "Blog"
+    text: "Real User Monitoring"
+  - link: "/real_user_monitoring/explorer/"
+    tag: "Documentation"
+    text: "Explore your views within Datadog"
+  - link: "/real_user_monitoring/explorer/analytics/"
+    tag: "Documentation"
+    text: "Build analytics upon your events"
+  - link: "/real_user_monitoring/dashboards/"
+    tag: "Documentation"
+    text: "RUM Dashboards"
+---
+
+When a user visits a page on your website, a new **RUM view event** is created. The view contains live page load and performance metrics. Performance metrics are sent to Datadog as the page finishes loading, but might be empty for a few seconds before they appear in the Datadog UI.
+
+## Identifying performance bottlenecks
+
+To find what's causing a page to under-perform:
+
+1. Set up and configure [RUM Browser monitoring][1] for your application.
+2. Open the [Performance Overview dashboard][2]. It provides you with a high level view of your monitored pages, and you can apply filters to find pages with performance problems.
+3. When you find a page with problem, click the event to open it in the [RUM Explorer][3].
```

Comment:
Let's mention:
1. the interaction with the widget
2. the jump from an overview to full granularity of data in the explorer. 

I want to really emphasize on the granularity => that's a key differentiator

Reply:
I think I need to see this in action to know what you're talking about enough to write about it. OR, maybe we could record a gif that shows it.

Reply:
https://a.cl.ly/BluQppQp

For instance in this GIF, we identify that a group is page is slow in dashboards and then we can look at individual loads of that page in the RUM explorer, look at all the resources loaded etc. Being able to jump from overview to details is key in our strategy :)

Reply:
Perfect. I will put this in words, but i also think I'll add a gif in the docs. This is great, thank you.